CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+## 1.11.0-beta.1
+
+### ✨ Features
+* Add scale option to markers, to support custom sizing when using the default Marker SVG element.([#9414](https://github.com/mapbox/mapbox-gl-js/pull/9414)) (h/t [adrianababakanian](https://github.com/adrianababakanian))
+
+### 🍏 Improvements
+* Improving the shader compilation speed by manually getting the run-time attributes and uniforms.([#9497](https://github.com/mapbox/mapbox-gl-js/pull/9497))
+* Mapbox gl-js documentation improvements.([#9607](https://github.com/mapbox/mapbox-gl-js/pull/9607))
+
+### 🐞 Bug fixes
+* Fix multi-finger gestures that touch markers.([#9683](https://github.com/mapbox/mapbox-gl-js/pull/9683))
+* Fix files within the style-spec package importing files from the parent repository tree.([#9522](https://github.com/mapbox/mapbox-gl-js/pull/9522))
```

Comment:
This one can be removed as well, already in 1.10 and 1.9.

Reply:
Couldn't find `9522` in the 1.10 and 1.9.. shouldn't the cl number be in the changelog already?

Reply:
This was probably skipped as well from the changelog, but it was published as part of the release.

Reply:
Actually I just checked this one and it was in the stylespec: https://github.com/mapbox/mapbox-gl-js/blob/master/src/style-spec/CHANGELOG.md#13131